config/core/test-configs.yaml

```
@@ -331,6 +331,11 @@ device_default_filters:
 
 device_types:
 
+  acer-mt8183-cp311-3h-jacuzzi:
```

```suggestion
  mt8183-kukui-jacuzzi-juniper-sku16:
```
As discussed on this PR.

---

updated